freqtrade/exchange/binance.py

```diff
@@ -23,13 +23,6 @@ class Binance(Exchange):
         "l2_limit_range": [5, 10, 20, 50, 100, 500, 1000],
     }
 
-    def stoploss_adjust(self, stop_loss: float, order: Dict) -> bool:
-        """
-        Verify stop_loss against stoploss-order value (limit or price)
-        Returns True if adjustment is necessary.
-        """
-        return order['type'] == 'stop_loss_limit' and stop_loss > float(order['info']['stopPrice'])
```

Comment:
The check for order-type was kindof intentional - as i don't really trust the exchange to return the correct order (especially binance in high-volume times ...).

If we think we can combine this to the exchange class, then please also look at  kucoin and huobi, which have very similar (if not identical) implementations.